test/navigation/OtherOfficialsNavigatorSpec.scala

```diff
@@ -254,6 +274,26 @@ class OtherOfficialsNavigatorSpec extends SpecBase {
         }
       }
 
+      "from the OtherOfficialsPreviousAddress Page" must {
+
+        "go to the SessionExpiredController page when user answer is empty" in {
+          navigator.nextPage(OtherOfficialsPreviousAddressPage(0), CheckMode, emptyUserAnswers) mustBe
+            routes.SessionExpiredController.onPageLoad()
+        }
+
+        "go to Previous Address Lookup flow when yes is selected and clicked continue button" in {
```

Comment:
these tests are not correct, could you please add one test  to redirect to deadend controller //TODO summary pages